src/Constants.js

```diff
@@ -33,7 +33,15 @@ var uniprotSources = [
         category: 'ANTIGEN'
     }
 ];
-var allSources = uniprotSources.slice(0);
+var otherSources = [
+    {
+        // To change later with /lrs/variants/
+        url: 'https://my-json-server.typicode.com/pconesa/demo/',
```

Comment:
@pconesa @jrmacias Can we merge this PR for 3.1.0? Should we change this url parameter or is this a placeholder for future code?